doc/misc/osi_vision.adoc

```diff
@@ -1,11 +1,6 @@
-= The vision of Open Simulation Interface
+= The idea of Open Simulation Interface
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu.
-Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a.
-Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+The Open Simulation Interface (OSI) is a specification for interfaces between models and components of a distributed simulation.
+OSI uses Google's Protocol Buffers.
+OSI has a strong focus on environmental perception of automated driving functions.
```

Comment:
```suggestion
OSI has a strong focus on environmental perception for automated driving functions.
```

Reply:
focus on environmental perception **through sensor models** for automated driving functions?
Just as suggestion (as we are always talking about sensor models it feels strange to exclude it here)